_posts/0000-01-14-jayasaluja.md

```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![fry](https://cloud.githubusercontent.com/assets/16547949/25400891/fb18941a-29c1-11e7-82b9-35e589443a60.jpg)
```

Comment:
@jayasaluja , you could technically modify the slide title to have it say something different when you slide is displayed in the slide show.